core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -952,6 +952,16 @@
    */
   public static final SqlAggFunction COUNT = new SqlCountAggFunction("COUNT");
 
+  /**
+   * <code>MODE</code> aggregate function.
+   */
+  public static final SqlAggFunction MODE =
+      SqlBasicAggFunction
+          .create("MODE", SqlKind.MODE,
+              ReturnTypes.ARG0_NULLABLE_IF_EMPTY, OperandTypes.ANY_NOT_NULL)
```

Comment:
In the JIRA discussion, I see that MODE allows NULL but if NULL is the most common value, return the second most common value instead. Why you choose to does not allow now in type?

Reply:
Yes. You are right. But This parameter means cant directly MODE(NULL). When using MODE(ColumnName) if the column value most common value is NULL, will return the second most common value insteed.

Reply:
Ah thanks for the clarification. Let me double check `OperandTypes.ANY_NOT_NULL`

Reply:
If no other problem about this, Please let me known or "Resolve conversation". Thank you.